package.json

```diff
@@ -62,6 +62,7 @@
 		"onCommand:cosmosDB.deleteDocDBDocument",
 		"onCommand:cosmosDB.deleteGraphDatabase",
 		"onCommand:cosmosDB.deleteGraph",
+		"onCommand:cosmosDB.importDocument",
```

Comment:
You should add this to the editor context menu as well:
![image](https://user-images.githubusercontent.com/6913354/46633100-8e715d00-cb01-11e8-956d-a5afee1918ef.png)

Reply:
See docker package.json for how to do this.